.github/workflows/build.yml

```
@@ -1,257 +1,316 @@
-        uses: ncipollo/release-action@v1
+      - name: Upload Release Assets
+        uses: softprops/action-gh-release@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          repo: dev-unstable
-          owner: emuflight
-          token: ${{ secrets.NC_PAT_EMUF }}
-          tag: "${{ env.NOW }}-cfg"
+          tag_name: ${{ github.ref_name }}
+          release_name: "EmuConfigurator ${{ github.ref_name }}"
```

actionlint (1.7.11)

[error] input "release_name" is not defined in action "softprops/action-gh-release@v2". available inputs are "append_body", "body", "body_path", "discussion_category_name", "draft", "fail_on_unmatched_files", "files", "generate_release_notes", "make_latest", "name", "overwrite_files", "prerelease", "preserve_order", "repository", "tag_name", "target_commitish", "token", "working_directory"

(action)